eq_translations/extractable_strings.py

```diff
@@ -0,0 +1,213 @@
+        "context": "Question",
+        "additional_context": "List",
+    },
+    {
+        "json_path": "$..question.calculation.title",
+        "description": "Question calculation title",
+        "context": "Question",
+    },
+    {
+        "json_path": "$..answers[*].validation.messages.*",
+        "description": "Answer error message",
+        "context": "Question",
+    },
+    {
+        "json_path": "$..answers[*].label",
+        "description": "Answer",
+        "context": "Question",
+    },
+    {
+        "json_path": "$..answers[*].description",
+        "description": "Answer description",
+        "context": "Question",
+        "additional_context": "Answer",
+    },
+    {
+        "json_path": "$..answers[*].playback",
+        "description": "Relationships playback template",
+        "context": "Question",
+    },
+    {
+        "json_path": "$..answers[*].options[*].label",
+        "description": "Answer option",
+        "context": "Question",
+    },
+    {
+        "json_path": "$..answers[*].options[*].description",
+        "description": "Answer option description",
+        "context": "Question",
+        "additional_context": "AnswerOption",
+    },
+    {
+        "json_path": "$..answers[*].options[*].detail_answer.label",
+        "description": "Detail answer label",
+        "context": "Question",
+        "additional_context": "AnswerOption",
+    },
+    {
+        "json_path": "$..answers[*].options[*].detail_answer.description",
+        "description": "Detail answer description",
+        "context": "Question",
+        "additional_context": "AnswerOption",
+    },
+    {
+        "json_path": "$..answers[*].options[*].title",
+        "description": "Relationships answer option question text",
+        "context": "Question",
+    },
+    {
+        "json_path": "$..answers[*].options[*].playback",
+        "description": "Relationships answer option playback text",
+        "context": "Question",
+    },
+    {
+        "json_path": "$..answers[*].guidance.show_guidance",
+        "description": "Answer guidance show link",
+        "context": "Question",
+    },
+    {
+        "json_path": "$..answers[*].guidance.hide_guidance",
+        "description": "Answer guidance hide link",
+        "context": "Question",
+    },
+    {
+        "json_path": "$..answers[*].guidance.contents[*].title",
+        "description": "Answer guidance heading",
+        "context": "Question",
+    },
+    {
+        "json_path": "$..answers[*].guidance.contents[*].description",
+        "description": "Answer guidance description",
+        "context": "Question",
+    },
+    {
+        "json_path": "$..answers[*].guidance.contents[*].list[*]",
+        "description": "Answer guidance list item",
+        "context": "Question",
+    },
+]
+
+CONTEXT_DEFINITIONS = {
+    "Question": {
+        "parent_schema_property": "question",
+        "context": {"property": "title", "text": "{context}"},
+    },
```

Comment:
Not sure it is clear that EXTACTABLE_STRING have `context` and `additional_context`, but then `context` and `additional_context` (CONTEXT_DEFINITIONS) has `context` and `secondary_context`. 

I get that context and additional_context are not mutually exclusive, whereas context and seconday_context are. It should really be `primary_content` and `secondary_context` but I understand that calling it `context` allows us to currently reuse the same method.

What if they were both called primary and secondary?. Otherwise, `context` should really be a list, as we had further context, we wouldn't want another key for it?

Hopefully what I said made sense if not, just msg :)

Reply:
I agree. The main reason the secondary context exists is so that we only output the list description or list title (if either exist). If having both in the output is acceptable, I think we can simplify this (while also making additional context more flexible).

At the moment we only allow a single `additional_context`, but that could be more as they just end up as additional comment lines in the generated pot file. If we: 
- changed `additional_context` to be a list
- separated the `List` context definition into `ListDescription` and `ListHeading`
- updated the existing list item string definitions to `"context": ["ListHeading", "ListDescription"]`

then list items would get both list title and list description comments in the pot file, which is both simpler and allows for more than one additional context for any extractable string.

Doing this will also mean we can drop the `context` property from the context definitions, so each definition would look something like:
```
"ListHeading": {
    "parent_schema_property": "contents",
    "property": "title", 
    "text": "For heading: {context}"},
},
```

Reply:
That sounds like the appropriate approach. 👍 
For your last bullet point, I assume you mean't `"additional_context": ["ListHeading", "ListDescription"]`?

Reply:
oh. yeah 😜